accountserver/sqlite_backend.go

```diff
@@ -26,6 +26,7 @@ import (
 	"io"
 	"io/ioutil"
 	"math"
+	"net/http"
```

Comment:
Someday we will have one Api that doesn't end up just returning http responses.

Reply:
we could make some error types or something. then do the err == NewerEntry or some crap for the 409 and otherwise 500.

Reply:
as a follow up :)